client/src/Header.scss

```diff
@@ -0,0 +1,15 @@
+@import '~bootstrap/scss/_functions.scss';
+@import '~bootstrap/scss/_variables.scss';
```

Comment:
Importing these bootstrap sass files lets you reference bootstrap variables like the $teal-400 color below.